htdocs/user/class/user.class.php

```diff
@@ -1483,9 +1483,9 @@ function update($user, $notrigger=0, $nosyncmember=0, $nosyncmemberpass=0, $nosy
 					// This user is linked with a member, so we also update member information
 					// if this is an update.
 					$adh=new Adherent($this->db);
-					$result=$adh->fetch('',$this->fk_member);
+					$result=$adh->fetch($this->fk_member);
 
-					if ($result > 0)
+					if ($result > 0 && $adh->entity == $conf->entity)
```

Comment:
@atm-gauthier @eldy 
i don't understand why ??
what happens if we use the centralization of users on the main entity with a sharing of the membership base?

Reply:
@hregis you're right sorry, problem i had was with tests on fetch functions